examples/vue/package.json

```diff
@@ -12,13 +12,13 @@
   },
   "dependencies": {
     "@aws-amplify/ui-vue": "^4.3.3",
-    "vue": "^3.0.5",
+    "vue": "^3.5.14",
     "vue-router": "4"
   },
   "devDependencies": {
     "@types/node": "18.8.0",
-    "@vue/compiler-sfc": "^3.0.5",
+    "@vue/compiler-sfc": "^3.5.14",
```

Comment:
Remove these changes, as a reviewer it makes me question whether the changes impact the build artifacts